packages/server/src/api/routes/tests/queries/generic-sql.spec.ts

```diff
@@ -1,26 +1,43 @@
-import { Datasource, Query } from "@budibase/types"
```

Comment:
very nice - cool to get these all into one test!

Reply:
Was a bit of a pain picking SQL that worked in all places 😄